### PR TITLE
Add New England

### PR DESCRIPTION
**Pull request by systemcatch**
Working parser for the New England region comprised of Connecticut, Maine, Massachusetts, New Hampshire, Rhode Island and Vermont.  Mapped generation to type as discussed with @corradio and @brunolajoie on slack.

I've also committed the new mockserver state.

new_england
new_england_breakdown

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -1488,6 +1489,24 @@
   "US-HI": {
     "flag_file_name": "us.png"
   },
+  "US-NEISO": {
+    "capacity": {
```
Comment (corradio) [inline config/zones.json]: neat!

**Comment by corradio** [comment]
Is that normal that I seem to have problems accessing https://www.iso-ne.com/ws/wsclient ? I get a timeout..

**Comment by systemcatch** [comment]
Yes I think that they implement some form of geo blocking for some countries outside the USA.  I can access it from the UK but they apparently block the Netherlands too.  Is that a problem for the server?

**Comment by systemcatch** [comment]
@corradio Ah sorry no I was thinking of another US state.  That link you posted only takes POST requests not GET.  The generation mix app on https://www.iso-ne.com/isoexpress/  posts to that link to retrieve it's data.

**Comment by corradio** [comment]
ok!

**Comment by HansHyde** [comment]
@systemcatch Neat, how'd you get the 6 state geometries as 1?

**Comment by systemcatch** [comment]
Modified countrytopos.js to merge the 6 into 1.  It's easy enough to make
regions out of the current natural earth set of shapes.

**Comment by HansHyde** [comment]
@systemcatch - shared files with you via Dropbox

The GoogleEarth .kmz file has NYISO connections to NEISO, RTD csv link & definitions for each string that it reports.  Github doesn't allow .kmz files. :(

3 NYISO connections to NEISO - "White Triangles" on Map
1. to NYISO 1  (NY geometry, NYISO existing parser)
2. to NYISO 2 (CT geometry, NYISO 2 parser)
3. to NYISO 3 (RI geometry, NYISO 3 parser)

Please remove CT & RI geometries from the NEISO

nyiso surrounding

@corradio @jarek fyi

**Pull request merged**